<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1225</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1030</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>1000</width>
     <height>581</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>30</y>
     <width>47</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>30</y>
     <width>47</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbNewGame">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Новая игра</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbBack">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditCoordinates">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>150</y>
     <width>191</width>
     <height>491</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier New</family>
    </font>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditInfo">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>60</y>
     <width>191</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier New</family>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="pbListen">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>20</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Listen</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
